### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by tuttinator**
Referencing to #6

This is my first approach at adding some tests. Not ready to merge yet, and I'll keep adding to this PR as I go - but I thought I'd open it early to allow you to provide feedback instead of one large PR all at once.

I've added a `Rakefile` as Travis CI will just run `rake` by default.

I've also thought about using [VCR](https://github.com/vcr/vcr) to stub HTTP requests - so that testing API calls don't hit real servers.

Feel free to let me know if there are any stylistic changes you prefer.

### CODE REVIEW

File: spec/document_cloud_spec.rb
Comment (tuttinator) [inline spec/document_cloud_spec.rb:17]: This isn't necessarily the nicest way to test this. Reaching in with `instance_variable_get` is something of an antipattern, and I'll try other approaches.

**Comment by mileszim** [comment]
Honestly maintenance of this is a low priority so whatever work you can contribute will be awesome. Feel free to use whatever tools you deem best, I trust you know what will work well since you've been using it frequently. I'm happy to merge when you think it's ready.

**Comment by mileszim** [comment]
Tests are passing @tuttinator, it's been far too long and I apologize getting back to you. Do you think this is merge ready?

File: spec/document_cloud_spec.rb
Comment (mileszim) [reply spec/document_cloud_spec.rb:17]: It's just a test, so if it works and handles the case the test is for, im good with it :)

**Review by mileszim — APPROVED** [review]
@tuttinator I'm going to work off this branch to get tests in before i merge this and start updating dependencies, is that alright with you?

**Comment by mileszim** [comment]
Great work! I borrowed your scaffold and am going from there. Thank you! Closing this in favor of #17

**Pull request closed without merging**
